.github/workflows/tag_release.yml

```diff
@@ -0,0 +1,59 @@
+name: tag-release
+
+on:
+  push:
+    # Pattern matched against refs/tags
+    tags:        
+      - '**' # This includes hierarchical tags
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+   
+jobs:
+  source-build:
+    uses: ./.github/workflows/source_build.yml
+
+  python-wheels:
+    needs: source-build
+    uses: ./.github/workflows/test_python.yml
+    with:
+      build_only: "true"
+
+  release-python:
+    needs: python-wheels
+    if: ( success() || failure() )
```

Comment:
Why does this continue on failure?

Reply:
Fixed - this was so I could continue despite macos wheels failing